app/src/main/java/com/douglasbruce/tasky/core/datastore/TaskyPreferencesDataSource.kt

```diff
@@ -0,0 +1,45 @@
+package com.douglasbruce.tasky.core.datastore
```

Comment:
I see, you have a package `data` and a package `datastore`. How comes, data store does not belong into `data`?

Reply:
An oversight, will move datastore into the data package